src/coreclr/pal/tests/palsuite/CMakeLists.txt

```diff
@@ -17,12 +17,14 @@ list(APPEND COMMON_TEST_LIBRARIES coreclrpal)
 if (CMAKE_CXX_COMPILER_ID MATCHES "Clang")
   add_compile_options(-Wno-incompatible-pointer-types-discards-qualifiers)
   add_compile_options(-Wno-int-to-void-pointer-cast)
+  add_compile_options(-Wno-format)
```

Comment:
nit: we can move this out of the conditional block so it applies to one.

For the future, can we add a `# TODO: fix formatting issues`?

Reply:
I'm not sure putting a lot of effort into fixing the PAL tests format violation is a good use of time. I don't like creating new conditionals in practice so avoided making another one. I didn't place it unconditionally because I assumed, perhaps wrongly, that these tests compile on Windows too. This means it would need to be conditioned on non-Windows.

Reply:
Yup, paltests project is Unix-only so it was just about moving it outside the conditions.